pandas/core/indexes/base.py

```diff
@@ -498,7 +495,7 @@ def __new__(
 
         klass = cls._dtype_to_subclass(arr.dtype)
 
-        # _ensure_array _may_ be unnecessary once Int64Index etc are gone
+        # _ensure_array _may_ be unnecessary once NumericIndex etc are gone
```

Comment:
i think changing this misses the point of this comment

Reply:
I'm not sure I follow you here, could you expand?

Reply:
the comment is suggesting that _ensure_array will become unnecessary